src/main/blackbox/blackbox_io.c

```diff
@@ -36,6 +36,7 @@
 #include "io/escservo.h"
 #include "rx/rx.h"
 #include "io/rc_controls.h"
+#include "io/osd.h"
```

Comment:
i had the exact same question when i was looking at this the first time. It's b/c config_master.h depends on the osd type. Remove it and:

```
%% blackbox_io.c
In file included from ./src/main/blackbox/blackbox_io.c:62:0:
./src/main/config/config_master.h:134:5: error: unknown type name 'osd_profile'
     osd_profile osdProfile;
     ^
```